src/xdocs/google_style.xml

```diff
@@ -685,7 +685,7 @@
                             <td><a name="4.6.2"/><a href="#4.6.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.6.2-horizontal-whitespace">4.6.2 Horizontal whitespace</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.6.2-horizontal-whitespace">4.6.2 Horizontal whitespace</a>
                             </td>
```

Comment:
@agcuda , please move all non-working cases to separate issues for further processing.
this PR already become to big to be maintainable.